aeron-client/src/main/c/aeron_windows.c

```diff
@@ -215,6 +215,21 @@ char *aeron_strndup(const char *value, size_t length)
     return dup;
 }
 
+int aeron_setenv(const char *name, const char *value, int overwrite)
```

Comment:
Unify these with the definitions in aeron_env.c/h

Reply:
@mikeb01 I removed my functions.